.github/workflows/release-build-tag-release.yml

```diff
@@ -11,11 +11,11 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [18.x]
+        node-version: ['20']
```

Comment:
Maybe 20.x ?

```suggestion
        node-version: [20.x]
```

Reply:
Is `node-version` even consumed in this workflow? If not, we should remove the matrix, right?

Reply:
You are right. So I removed it.